prometheus-to-sd/main.go

```diff
@@ -228,3 +240,18 @@ func readAndPushDataToStackdriver(stackdriverService *v3.Service, gceConf *confi
 		timeSeriesBuilder.Update(metrics, scrapeTimestamp)
 	}
 }
+
+func getNodeLabels(nodeLabelsStr string) map[string]string {
+	labels := make(map[string]string)
+	m, err := url.ParseQuery(nodeLabelsStr)
+	if err != nil {
+		glog.Fatalf("Error parsing 'node-labels-override' field: '%v'.", nodeLabelsStr)
+	}
+	for k, v := range m {
+		if len(v) != 1 {
+			glog.Fatalf("Key '%v' in 'node-labels-override' doesn't have exactly one value.", k)
```

Comment:
Maybe worth printing all the values in addition to the key?

Reply:
Done.